components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -82,7 +87,10 @@ const WorkspacesPage: FunctionComponent = () => {
 
     return (
         <>
-            <Header title="Workspaces" subtitle="Manage recent and stopped workspaces." />
+            <Header
+                title="Workspaces"
+                subtitle="Manage, start and stop your personal development environments in the cloud."
```

Comment:
nit: explicitly mentioning starting and stopping seems redundant if we're already saying you can manage them.

Reply:
I've picked it from Figma Designs, will check it once with @phimae

Reply:
Looks like it was a old one (which was outdated in Figma), reverted this change in [`7008c69`](https://github.com/gitpod-io/gitpod/pull/20021/commits/7008c696da75a5cb82cddd3e9e3da3194de53b69)